packages/amplication-client/src/Purchase/FAQ.tsx

```diff
@@ -19,115 +19,118 @@ export const FAQ = () => {
+              and coding patterns.
+              <br />
+              <br />
+              We provide guides on how to add actions to REST API controllers,
+              add custom DTOs (Data Transfer Objects), add queries to GraphQL
+              resolver, and more on our documentation.
             </div>
           }
         />
         <Question
-          question="What happens when I downgrade a subscription?"
-          answer="You may downgrade your paid subscription any time before the renewal date. The subscription will automatically downgrade at the end of the billing cycle."
-        />
-        <Question
-          question="Are you going to change your pricing in the future?"
-          answer="The current pricing is set to show our appreciation to early adopters of our service. We may change the pricing in the future, however as a team of developers we are committed to pricing being as developer-friendly as possible. "
-        />
-        <Question
-          question="Are there any hidden fees or charges not included in the listed price?"
-          answer="The listed prices include all you need to generate your code. Amplication adds no extra charges.  However, in some countries or jurisdictions, additional VAT or other taxes may be payable (see below). "
-        />
-        <Question
-          question="What about VAT or other taxes?"
-          answer="All prices exclude tax, because there are different taxes applied in different areas. When paying by credit card, local taxes will be added upon subscription, in accordance with the details you'll provide. "
-        />
-        <Question
-          question="What forms of payment do you accept?"
-          answer="We accept payments via all major Credit Cards"
+          question="Do you support more languages besides TypeScript and Node.js?"
+          answer="Right now Amplication is focused on helping you generate production-ready Node.js backends. At the moment we don't support other languages."
         />
         <Question
-          question="Do I get a notification if I am approaching my usage limits?"
-          answer={`We constantly show you your plan limitations within your workspace. We will not block you from configuring your services but in order to generate the code you will need to comply with the workspace plan limits.
-
-          We will notify you via email or other means when we think it might be a good time for you to consider upgrading your workspace plan.`}
-        />
-        <Question
-          question="What if I need to add more team members?"
-          answer="No problem. Select the plan that meets your needs. The Free edition supports up to two user seats, the Pro edition supports up to five user seats, and the Enterprise edition has no limit. "
+          question="We have legacy systems, and would like to progress to modern architecture. Can you assist with it?"
+          answer={
```

Comment:
I like this answer, but I think it's too long. Maybe we can just mention the feature names, and link it to its documentation. WDYT?